corehq/apps/reminders/templates/reminders/partial/scheduled_reminders.html

```diff
@@ -151,6 +98,8 @@
 {% endblock %}
 
 {% block page_content %}
+    {% initial_page_data 'reminder_data' reminder_data %}
```

Comment:
@orangejenny @gcapalbo

I traced the recent bouts of soft asserts we just started getting a few days talkin bout "json_format_datetime expects offset-naive" back to this.

```
date: "{{ data.next_fire.date }}",		
time: "{{ data.next_fire.time }}",
```

does something subtly different because `next_fire` is [intentionally converted to a timezone-aware datetime](https://github.com/dimagi/commcare-hq/blob/96a03407d7572a1afb77398b65d7d3ba1fa478bc/corehq/apps/reminders/views.py#L130-130) for the purpose of showing the user time they way they're used to. The lines that were removed (I believe) use django's defaults to convert dates and times (with timezone?) for display to the user.

I'm not totally sure what users are seeing right now, but I'm guessing it's off from what's intended.